### ISSUE TITLE
flowbite version 1.5.5: CSP problem

### ISSUE THREAD

**Issue by PhilFlash** [themesberg/flowbite#359]
**Describe the bug**
With version 1.5.5 , i have a problem with CSP (Content Security Policy): 
```
Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' fonts.googleapis.com". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution.
```
With the same code and flowbite version 1.5.4, all is ok (no problem with CSP).

**To Reproduce**
I have a simple application (just header and slider, no javascript, just 'import flowbite'). I use webpack (with laravel-mix)
With version 1.5.4, all is ok.
Example: https://inwayvideo.com/flowbitecsp1/

With the same code, I just change version of flowbite with version 1.5.5
There is a problem with CSP.
```
Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' fonts.googleapis.com". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution.
```
Example: https://inwayvideo.com/flowbitecsp2/
Use chrome and open console to show message

Code is here (in a zip file):
http://inwayvideo.com/flowbitecsp/flowbiteBugCSP.zip
read readme.txt to build demo

**Expected behavior**
No message for CSP 

**Desktop (please complete the following information):**
 - Browser: same problem with chrome 108 / edge 108 / firefox 108
 - Version of flowbite : version 1.5.5

Philippe

**Comment by zoltanszogyenyi** [comment]
Hey @PhilFlash,

Thanks for the detailed report. It's an odd issue, but I'll fix it for the upcoming v1.6 release which also introduces TypeScript.

**Comment by zoltanszogyenyi** [comment]
Hey @PhilFlash,

We have released v1.6.0 - can you please check this out? If the issue still persists we'll provide a patch shortly.

**Comment by PhilFlash** [comment]
Hi,
All is ok with 1.6.0
Thanks
